lib/tasks/rspec.rake

```diff
@@ -13,6 +13,6 @@ namespace :knapsack_pro do
     ENV.delete('SPEC_OPTS')
 
     detector = KnapsackPro::TestCaseDetectors::RSpecTestExampleDetector.new
-    detector.generate_json_report
+    detector.generate_json_report(ENV['KNAPSACK_PRO_RSPEC_OPTIONS'])
```

Comment:
Should we raise if the ENV is not set?

Reply:
No. It's fine if people do not pass any args to the Knapsack Pro command like:

```
bundle exec rake knapsack_pro:queue:rspec
```

ENV['KNAPSACK_PRO_RSPEC_OPTIONS'] is nil then, and this is fine.